local-dev.yml

```diff
@@ -68,8 +88,8 @@ services:
       - "/dev/fuse"
     privileged: true
     environment:
-      POSTGIS_HOST: postgis
-      POSTGIS_PORT: 5432
+      POSTGIS_HOST: pgbouncer
+      POSTGIS_PORT: 6432
```

Comment:
please evaluate whether we should also make these changes on the `defaultworker` service, thanks